test/e2e/networks/upgrade.toml

```
@@ -0,0 +1,44 @@
+ipv6 = false
```

```suggestion
ipv6 = false
```
default value, no need to appear